packages/frontapp/src/app/components/BackButton/index.tsx

```diff
@@ -7,14 +7,15 @@ import { useNavigate } from 'react-router-dom';
 import { t } from 'i18next';
 
 export const ButtonWrapper = styled.div`
-  margin: 3rem 0 ${dimensions.xl_3} 0;
+  margin: ${dimensions.base_3} 0 ${dimensions.xl_3} 0;
   display: flex;
   align-items: center;
   cursor: pointer;
 `;
 export const GoBack = styled.a`
   font-weight: 600;
   font-size: ${dimensions.base};
+  margin-left: ${dimensions.lg};
```

Comment:
Can't we just use `gap` on the wrapper element instead of margin?

Reply:
Yeah, I think we can open a space by giving gap to the wrapper. Should I change it or it is ok?

Reply:
@mabrur-h yes change it, if it will work the same way as margin-left does